# add timestamp
std_msgs/Header header

float64 fpga_time

uint64[] modem_times